clustercontroller-core/src/test/java/com/yahoo/vespa/clustercontroller/core/restapiv2/StateRestApiTest.java

```diff
@@ -57,8 +57,8 @@ protected void setUp(boolean dontInitializeNode2) throws Exception {
             Set<ConfiguredNode> nodesInSlobrok = FleetControllerTest.toNodes(1, 3, 5, 7);
 
             ContentCluster cluster = new ContentCluster(
-                    "music", nodes, distribution, 4 /* minStorageNodesUp*/, 0.0 /* minRatioOfStorageNodesUp */,
-                    true /* determineBucketsFromBucketSpaceMetric */);
+                    "music", nodes, distribution, 4 /* minStorageNodesUp*/, 0.0 /* minRatioOfStorageNodesUp */
+                    /* determineBucketsFromBucketSpaceMetric */);
```

Comment:
Remove comment for the removed argument as well

Reply:
Fixed